src/router/router.ts

```diff
@@ -29,12 +29,6 @@ export function getVueRouter () {
         name: 'signin',
         query: { redirect: to.fullPath }
       })
-    } else if (!isSigninRoute(to) && !isSignoutRoute(to) && !haveQueryParam(to.query)) {
-      // for normal routes, re-route along with query params
-      next({
-        name: to.name,
-        query: { ...from.query, ...to.query }
-      })
```

Comment:
Did you test this change with Corrections?

Reply:
Yea, corrections doesn't ever catch in this block because we don't enter corrections without a query param.
Also removed the `corrected-id` because we are always entering with a draft. ie `correction-id`.

Reply:
Could certainly add in another check here, though i'm not sure what at this point.

Reply:
What happens if we do enter a correction without a query param?

Also did you try loading an alteration and a correction while NOT first logged in. It should redirect you to login and the return here. (Might be tricky to test with IDIR login, sigh.)

Reply:
For sure!

If we try to enter correction without a query param, it errors out while trying to load the app. Get's caught on the filingId. 
If we have this code block in however, it gets caught in a router error which is somewhat of a loop `Maximum Call stack exceeds..` etc etc. 

This is also the same for Alterations without a query param with this block in.

If we try and enter either without a token in session, it directs us to Auth, and returns us the route of origin as expected.

Short answer, i think in the present state we are ok without this check for a query param BUT we should handle the error where no query param is present or invalid for CORRECTIONS.